package.json

```diff
@@ -115,6 +115,7 @@
   "dependencies": {
     "@moduk/frontend": "^0.4.1",
     "core-js": "^3.27.2",
-    "iframe-resizer": "^4.3.3"
+    "iframe-resizer": "^4.3.3",
+    "node-html-parser": "^6.1.5"
```

Comment:
Did you mean to move this from devDependencies? Should be only be being used at build-time

Reply:
oh no